frontend/packages/console-dynamic-plugin-sdk/src/runtime/plugin-loader.ts

```diff
@@ -148,46 +144,16 @@ export const registerPluginEntryCallback = (pluginStore: PluginStore) => {
   window.loadPluginEntry = getPluginEntryCallback(pluginStore, resolveEncodedCodeRefs);
 };
 
-export const adaptPluginManifest = (
```

Comment:
:+1: this simplifies our plugin manifest code and will make it easier to use `PluginStore` and `PluginLoader` from OpenShift plugin SDK.